tensorboard/backend/event_processing/event_accumulator.py

```diff
@@ -81,17 +82,15 @@
     "tensor": "_ProcessTensor",
 }
 
-## The tagTypes below are just arbitrary strings chosen to pass the type
-## information of the tag from the backend to the frontend
-COMPRESSED_HISTOGRAMS = "distributions"
-HISTOGRAMS = "histograms"
-IMAGES = "images"
-AUDIO = "audio"
-SCALARS = "scalars"
-TENSORS = "tensors"
-GRAPH = "graph"
-META_GRAPH = "meta_graph"
-RUN_METADATA = "run_metadata"
+COMPRESSED_HISTOGRAMS = tag_types.COMPRESSED_HISTOGRAMS
```

Comment:
Maybe add a comment like "Legacy aliases" or something?  Just to indicate why we're copying them all.

Reply:
Done.